Source/LuaBridge/detail/LuaRef.h

```diff
@@ -518,12 +518,34 @@ class LuaRefBase
     {
         impl().push();
         pushArguments(std::forward<Arguments>(arguments)...);
-        LuaException::pcall(m_L, sizeof...(arguments), 1);
+        int nargs = sizeof...(arguments);
+        int msgh = lua_gettop(m_L) - nargs;
+        lua_pushcfunction(m_L, msghandler);
+        lua_insert(m_L, msgh);
+        LuaException::pcall(m_L, nargs, 1, msgh);
+        lua_remove(m_L, msgh);
```

Comment:
This change doesn't look like a compiler error fix.

Reply:
wrong branch.